client/asset/btc/btc_test.go

```diff
@@ -391,13 +391,14 @@ func tNewWallet(segwit bool) (*ExchangeWallet, *tRPCClient, func()) {
 	}
 	walletCtx, shutdown := context.WithCancel(tCtx)
 	cfg := &BTCCloneCFG{
-		WalletCFG:          walletCfg,
-		Symbol:             "btc",
-		Logger:             tLogger,
-		ChainParams:        &chaincfg.MainNetParams,
-		WalletInfo:         WalletInfo,
-		DefaultFallbackFee: defaultFee,
-		Segwit:             segwit,
+		WalletCFG:           walletCfg,
+		Symbol:              "btc",
+		Logger:              tLogger,
+		ChainParams:         &chaincfg.MainNetParams,
+		WalletInfo:          WalletInfo,
+		DefaultFallbackFee:  defaultFee,
+		DefaultFeeRateLimit: defaultFeeRateLimit,
```

Comment:
You'll need to do this in **client/asset/ltc/ltc.go** too. Might be worth investigating why no tests caught that.

Reply:
Added